susemanager-utils/testing/docker/scripts/cobbler/setup-reposync.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Dependencies for reposync tests
+zypper install --no-recommends -y dnf python3-librepo dnf dnf-plugins-core wget \
+       perl-LockFile-Simple perl-Net-INET6Glue perl-LWP-Protocol-https ed
+dnf install -y  http://download.fedoraproject.org/pub/fedora/linux/releases/35/Everything/x86_64/os/Packages/d/debmirror-2.35-2.fc35.noarch.rpm
```

Comment:
Will this be around for a long time? Today is the last date of Fedora 35 support (https://ask.fedoraproject.org/t/fedora-37-released-fedora-35-reaches-end-of-life-in-december-22/28538)

Reply:
These setup scripts were simply taken from upstream Cobbler, I don't really know which was the criteria when they were implemented.

OTOH, about Fedora 35 EOL and availability of repositories, from what I can see it seems they keep the repository published for some months before moving it to "archive". I think we should change this, at least, to a more recent Fedora version.

Reply:
> These setup scripts were simply taken from upstream Cobbler, I don't really know which was the criteria when they were implemented.

How do we make sure we don't diverge if we just want to consume them from upstream? Are they not packaged in any way for us to consume in our downstream tests? Could we contribute a test or test-support subpackage to upstream that we just consume?

We break "links" to upstream sources all the time, why not take advantage of the maintenance that's being done upstream.

Reply:
Thanks for the suggestions Alex. Finally I have this implemented:

- new `cobbler-tests-containers` subpackage containing setup/configuration files to run Cobbler tests in containers.
- consume `cobbler-tests-containers` package from our CI containers.

Besides of this PR, these are the necessary changes from the Cobbler side:

- Upstream Cobbler PR: https://github.com/cobbler/cobbler/pull/3302
- Uyuni:Master SR: https://build.opensuse.org/request/show/1043335 (backported also to IBS)
- SRs created also for "D:G:M:4.3" and "D:G:M:Head"